src/test/ui/parser/macro/trait-object-macro-matcher.rs

```diff
@@ -10,5 +10,6 @@ macro_rules! m {
 fn main() {
     m!('static);
     //~^ ERROR lifetime in trait object type must be followed by `+`
+    //~| ERROR lifetime in trait object type must be followed by `+`
```

Comment:
This is understandable.
`'static` is now parsed as a type twice, both times in prints this error and recovers.